content/hardware/04.pro/boards/portenta-c33/tutorials/user-manual/content.md

```diff
@@ -1426,6 +1426,174 @@ Here's what each section of the example code does:
+
+
+![Arduino IoT Cloud Setup Device pop-up window](assets/user-manual-17.png) 
+
+Once the Portenta C33 has been configured, let's create a "Thing" to test the connection between your board and the Arduino IoT Cloud. Navigate into **Things** and select the **CREATE THING** button; give your thing a name. 
+
+
+![Arduino IoT Cloud "Thing" setup](assets/user-manual-18.png) 
+
+Navigate into **Associate Device** and click the **Select Device** button, select your Portenta C33 board and associate it to your "Thing." Then, navigate into **Network** and click the **Configure** button; enter your network credentials. The project is now ready to add some variables to your "Thing"; navigate into **Cloud Variables** and click the **ADD VARIABLE** button. 
+
+![Add variable button](assets/user-manual-19.png) 
+
+
+Add one variable with the following characteristics:
+
+- **Name**: `led`
+- **Variable type**: `boolean`
+- **Variable permission** `Read & Write`
+- **Variable update policy**: `On change`
+
+![Arduino IoT Cloud "Thing" variable setup](assets/user-manual-21.png) 
+
+You should see the `led` variable in the **Cloud Variables** section. Navigate into **Dashboards** and select the **BUILD DASHBOARD** button; this will create a new dashboard; give your dashboard a name. 
+
+Add the following widgets to your dashboard:
+
+- **Switch**: name the widget `Switch` and link it to the `led` variable you created before. 
+- **LED**: name the widget `led` and link it to the `led` variable you created before. 
+
+Your dashboard should see the following:
+
+![Arduino IoT Cloud Dashboard setup](assets/user-manual-20.png)
+
+Go back to your **Things** and open the "Thing" you created. In the "Thing" setup page, navigate into **Sketch**, where you should see the online editor. 
+
+
+In the generated sketch, define `LED_BUILTIN` pin as an output in the `setup()` function:
+
+```arduino
+void setup() {
+  // Initialize serial and wait for port to open:
+  Serial.begin(9600);
+  // This delay gives the chance to wait for a Serial Monitor without blocking if none is found
+  delay(1500);
+
+  // LED_BUILTIN macro access the onboard green LED 
+  pinMode(LED_BUILTIN, OUTPUT);   
+
+  // Defined in thingProperties.h
+  initProperties();
+
+  // Connect to Arduino IoT Cloud
+  ArduinoCloud.begin(ArduinoIoTPreferredConnection);
+  
+  /*
+     The following function allows you to obtain more information
+     related to the state of network and IoT Cloud connection and errors
+     the higher number the more granular information you’ll get.
+     The default is 0 (only errors).
+     Maximum is 4
+ */
+  setDebugMessageLevel(2);
+  ArduinoCloud.printDebugInfo();
+}
+```
+
+In the `onLedChange()` function, you must set the LED state with the `led` variable.

```

Comment:
I'd suggest to add somewhere that this function is automatically generated for the user by the cloud... Like a low code approach, what do you think? cc @jcarolinares

Reply:
What about this @martab1994?
![image](https://github.com/arduino/docs-content/assets/20689211/234e0bad-210b-4146-85fc-030b3c23eb7c)

Reply:
@canchebagur small improvement IMO

In the onLedChange() function, which was generated automatically when the cloud variable led was created, you must vinculate the LED state to the led variable.

Remember that the screenshot that you shared with me does not have the NOT !led so be careful

Reply:
"Vinculate" sounds weird and seems not to exist or not to be used frequently; change for "associate."

Reply:
Fine for me thanks 👍